src/main/kotlin/no/nav/syfo/App.kt

```diff
@@ -84,11 +84,11 @@ fun main() {
             )
             dialogmeldingStatusService = DialogmeldingStatusService(
                 database = applicationDatabase,
+                dialogmeldingToBehandlerService = dialogmeldingToBehandlerService,
```

Comment:
Kan godt være dette virker, men ser litt mindre rart ut dersom de to service-instansene bytter plass.